app/js/lib/foxbox.js

```diff
@@ -295,15 +297,12 @@ export default class Foxbox extends Service {
    * @return {Promise}
    */
   getServiceState(id) {
-    return new Promise((resolve, reject) => {
-      fetchJSON(`${this.origin}/services/${id}/state`)
-        .then(res => {
-          if (!res) {
-            return reject(new Error(`The action couldn't be performed.`));
-          }
+    return fetchJSON(`${this.origin}/services/${id}/state`).then(res => {
```

Comment:
Yeah, but ```resp.ok()``` is already checked in ```fetchJSON``` and if ```!resp``` is checked here (we throw - it's the same as ```reject```)?